jetty-client/src/main/java/org/eclipse/jetty/client/http/HttpReceiverOverHTTP.java

```diff
@@ -246,7 +246,7 @@ protected boolean isShutdown()
     public int getHeaderCacheSize()
     {
         // TODO get from configuration
-        return 4096;
+        return 1024;
```

Comment:
Can you please do this here:
```java
HttpClientTransportOverHTTP transport = (HttpClientTransportOverHTTP)getHttpDestination().getHttpClient().getTransport();
return transport.getHeaderCacheSize();
```
Then add getter/setter for `headerCacheSize` in `HttpClientTransportOverHTTP`.